readme.md

```diff
@@ -15,3 +15,5 @@
 ## Presentation
 
 To build the presentation run `make` and then open `index.html` in your browser.
+You will need `pandoc' installed, your friendly neighborhood package manager
```

Comment:
Would you make this one long line and change the trailing apostrophe to a backtick so that the markdown renders properly?

Reply:
I actually didn't notice you had a trailing back tick, silly me I assumed the old (and obtuse) custom...